compiler/luci/import/src/Nodes/CircleGreater.cpp

```diff
@@ -39,8 +46,18 @@ bool CircleGreaterGraphBuilder::validate(const ValidateArgs &args) const
   if (tensors[inputs[0]]->type != tensors[inputs[1]]->type)
     return false;
 
+  // NOTE: real models do have output dtype NOT BOOL
```

Comment:
Sentence is little bit confusing... @.@ Which one of them?

- Real models often have output dtype, which is NOT BOOL.
- Real models actually do not have output dtype, which is NOT BOOL

Reply:
> do have

I thought output dtype should be boolean, but it is possible that models can have not boolean. So. `do have` is emphasis  on this.